mmseg/models/backbones/vit.py

```diff
@@ -330,10 +299,17 @@ def init_weights(self, pretrained=None):
             else:
                 state_dict = checkpoint
 
+            if 'rwightman/pytorch-image-models' in pretrained:
```

Comment:
If user loaded the weight from timm and would like to init the model with path, the condition does not hold.